packages/pie-toolbox/src/code/mask-markup/components/blank.jsx

```diff
@@ -91,17 +77,32 @@ export class BlankContent extends React.Component {
       if (!currentChoice) {
         this.setState({
           height: 0,
+          width: 0,
         });
         return;
       }
       setTimeout(() => {
-        this.setState({
-          height: this.spanRef.offsetHeight,
-        });
+        this.updateDimensions();
       }, 300);
     }
   }
 
+  updateDimensions() {
+    const height = this.spanRef.offsetHeight;
+    const width = this.spanRef.offsetWidth;
+
+    // force our computing to work if our client set the emptyResponseAreaHeight
+    // both to '40px' or 40
+    const responseAreaHeight = parseFloat(this.props.emptyResponseAreaHeight);
+    const responseAreaWidth = parseFloat(this.props.emptyResponseAreaWidth);
+    if (height > responseAreaHeight || width > responseAreaWidth) {
```

Comment:
Sanity check to ensure that responseAreaHeight and responseAreaWidth are valid numbers after parsing.